doc/en/py27-py34-deprecation.rst

```diff
@@ -17,9 +17,9 @@ are available on PyPI.
 
 While pytest ``5.0`` will be the new mainstream and development version, until **January 2020**
 the pytest core team plans to make bug-fix releases of the pytest ``4.6`` series by
-back-porting patches to the ``4.6-maintenance`` branch that affect Python 2 users.
+back-porting patches to the ``4.6.x`` branch that affect Python 2 users.
```

Comment:
Since there is not an actual `4.6-maintenance` branch, I changed it for what I believe is correct.

Reply:
huh, this should be in master as well 🤔. I can create a PR for that too if I get confirmation that that's the right thing to do.

Reply:
No need to port to master, we have changed that significantly already: https://docs.pytest.org/en/latest/py27-py34-deprecation.html